File: SimpleUser.php
<?php

namespace App\Entity;

use App\Repository\SimpleUserRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;

/**
 * @ORM\Entity(repositoryClass=SimpleUserRepository::class)
 * @UniqueEntity("mail")
 */
class SimpleUser
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="il faut remplir ce champ")
     */
    private $nom;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="il faut remplir ce champ")
     */
    private $prenom;

    /**
     * @ORM\Column(type="integer")
     * @Assert\NotBlank(message="il faut remplir ce champ")
     * @Assert\Length(min="8",
     *     max="8",
     *     minMessage="Doit contenir {{ limit }} caractères",
     *     maxMessage="Doit contenir {{ limit }} caractères"
     * )
     */
    private $cin;

    /**
     * @ORM\Column(type="string", length=255, unique=true)
     * @Assert\NotBlank(message="il faut remplir ce champ")
     * @Assert\Email(message="Email {{ value }} non valide")
     */
    private $mail;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="il faut remplir ce champ")
     * @Assert\Length(min="8",
     *     max="20",
     *     minMessage="Doit contenir {{ limit }} caractères",
     *     maxMessage="Doit contenir {{ limit }} caractères"
     * )
     */
    private $mdp;

    /**
     * @ORM\Column(type="string", length=255, nullable=true)
     */
    private $block;

    /**
     * @ORM\Column(type="string", length=255, nullable=true)
     */
    private $codem;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getPrenom(): ?string
    {
        return $this->prenom;
    }

    public function setPrenom(string $prenom): self
    {
        $this->prenom = $prenom;

        return $this;
    }

    public function getCin(): ?int
    {
        return $this->cin;
    }

    public function setCin(int $cin): self
    {
        $this->cin = $cin;

        return $this;
    }

    public function getMail(): ?string
    {
        return $this->mail;
    }

    public function setMail(string $mail): self
    {
        $this->mail = $mail;

        return $this;
    }

    public function getMdp(): ?string
    {
        return $this->mdp;
    }

    public function setMdp(string $mdp): self
    {
        $this->mdp = $mdp;

        return $this;
    }

    public function getBlock(): ?string
    {
        return $this->block;
    }

    public function setBlock(?string $block): self
    {
        $this->block = $block;

        return $this;
    }

    public function getCodem(): ?string
    {
        return $this->codem;
    }

    public function setCodem(?string $codem): self
    {
        $this->codem = $codem;

        return $this;
    }
}

File: SimpleUser.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

/**
 *
 * @author mouad
 */
public class SimpleUser {
    private int id,cin;
    private String nom, prenom,mail, mdp, block, codem;

    public SimpleUser(String mail, String mdp) {
        this.mail = mail;
        this.mdp = mdp;
    }

    
    
    public SimpleUser(int id, int cin, String nom, String prenom, String mail, String mdp, String block, String codem) {
        this.id = id;
        this.cin = cin;
        this.nom = nom;
        this.prenom = prenom;
        this.mail = mail;
        this.mdp = mdp;
        this.block = block;
        this.codem = codem;
    }

    public SimpleUser(int cin, String nom, String prenom, String mail, String mdp, String block, String codem) {
        this.cin = cin;
        this.nom = nom;
        this.prenom = prenom;
        this.mail = mail;
        this.mdp = mdp;
        this.block = block;
        this.codem = codem;
    }

    public SimpleUser(int cin, String nom, String prenom, String mail, String mdp) {
        this.cin = cin;
        this.nom = nom;
        this.prenom = prenom;
        this.mail = mail;
        this.mdp = mdp;
    }

    public SimpleUser() {
    }
    
    

    
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getCin() {
        return cin;
    }

    public void setCin(int cin) {
        this.cin = cin;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public String getMail() {
        return mail;
    }

    public void setMail(String mail) {
        this.mail = mail;
    }

    public String getMdp() {
        return mdp;
    }

    public void setMdp(String mdp) {
        this.mdp = mdp;
    }

    public String getBlock() {
        return block;
    }

    public void setBlock(String block) {
        this.block = block;
    }

    public String getCodem() {
        return codem;
    }

    public void setCodem(String codem) {
        this.codem = codem;
    }

    @Override
    public String toString() {
        return "SimpleUser{" + "id=" + id + ", cin=" + cin + ", nom=" + nom + ", prenom=" + prenom + ", mail=" + mail + ", mdp=" + mdp + ", block=" + block + ", codem=" + codem + '}';
    }
    
    
    
}
